### PR TITLE
[14.0][MIG] contract_comment

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Standard Migration

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ connector
 account-invoicing
 account-financial-tools
 l10n-spain
+contract
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt:6]: Can you split the change to oca_dependencies in a different commit?

Comment (ChrisOForgeFlow) [reply oca_dependencies.txt:6]: Done

**Review by mariadforgeflow — APPROVED** [review]
LGTM!
